### PR TITLE
[OSDEV-1098] Add migration for the report "Contributor type by %"

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-1098](https://opensupplyhub.atlassian.net/browse/OSDEV-1098) Reporting. A columns values in the report "Contributor type by %" are not cumulative.

* Added migration for dropping unnecessary calc_column function and creation add_percent_to_number function for  the "Contributor type by %" report.

### CODE REVIEW

File: src/django/sqls/drop_calc_column_func.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS calc_column(numeric, numeric);
```
Comment (vladsha-dev) [inline src/django/sqls/drop_calc_column_func.sql:1]: There is no such function. Please use the actual types while deleting this function.

File: src/django/sqls/0150_revert_add_percent_to_number.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS add_percent_to_number(int8, int8);
```
Comment (vladsha-dev) [inline src/django/sqls/0150_revert_add_percent_to_number.sql:1]: If you want to delete this function, you should use the same parameter types that were set when declaring the function.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: src/django/sqls/0150_revert_add_percent_to_number.sql (hunk 2 above)
Comment (Innavin369) [reply src/django/sqls/0150_revert_add_percent_to_number.sql:1]: changed

**Review by vladsha-dev — DISMISSED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**